arcade/examples/sprite_depth_cosine.py

```diff
@@ -88,11 +89,26 @@ def on_key_press(self, symbol: int, modifiers: int):
             self.text_use_depth.text = f"SPACE: Toggle depth testing ({self.use_depth})"
 
     def on_update(self, delta_time):
-        self.time += delta_time
-
+        # We use absolute time instead of the delta, so we get it from the window's clock
```

Comment:
I would still change this to 'elapsed' since that's the language used by the clocks. It is also missing a 'the'

'We use the elapsed time instead of the delta, so we get it from the window's clock'

Reply:
Just use the time from the clock. There are several examples can could need this change.